.travis.yml

```diff
@@ -1,20 +1,23 @@
 language: go
 
 go:
-- 1.8
+  - 1.8
 
 services:
-- postgresql
+  - postgresql
+  - mysql
 
 before_install:
-- go get github.com/onsi/ginkgo/ginkgo
+  - go get github.com/onsi/ginkgo/ginkgo
 
 install:
-# Prevent default install task that does a `go get -t ./...`
-- true
+  # Prevent default install task that does a `go get -t ./...`
+  - true
 
 before_script:
   - psql -c 'create database mydb;' -U postgres
+  - mysql -e 'CREATE DATABASE template1;'
```

Comment:
What's this database used for? This looks like a Postgres specific thing. I'm not clear why it's needed for Mysql.